vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp

```diff
@@ -77,21 +84,41 @@ $( document ).ready(function() {
 			// TODO: show success/error message
 		});
 		$("#bgImage").on("click", function(e){});
+		$("#createSpaceLinkAlert").hide();	
+	});
+	
+	$('#spaceLinkRotation').change(function() {
+		$('#arrow').css('transform', 'rotate(' +$('#spaceLinkRotation').val()+ 'deg)');
+	});
+		
+	$('#changeBgImgButton').click(function(file) {
 		$("#createSpaceLinkAlert").hide();
+		$("#changeBgImgAlert").show();	
+		
 	});
+			
+	$("#changeBgImgAlert").draggable();
+		
+	$('#spaceLinkCreationModal.draggable>.modal-dialog>.modal-content>.modal-header').css('cursor', 'move');
 	
 	$('#spaceLinkRotation').change(function() {
 		$('#arrow').css('transform', 'rotate(' +$('#spaceLinkRotation').val()+ 'deg)');
 	});
+		$('#errorMsg').delay(2000).fadeOut();
 });
 </script>
 
  <h1>Space: ${space.name}</h1> 
-
+ 
 <div class="alert alert-light" role="alert">
   Created on <span class="date">${space.creationDate}</span> by ${space.createdBy}.
   <br>
-  Modified on <span class="date">${space.modificationDate}</span> by ${space.modifiedBy}.
+  Modified on <span class="date">${space.modificationDate}</span> by ${space.modifiedBy}. 
+  <c:if test="${param.showAlert eq true}">
+  	<div id=errorMsg class="alert alert-${param.alertType}">
+		<h3><strong>ERROR : ${param.message}</strong></h3>
```

Comment:
let's just make that simple text (not heading size and bold). but the alert dialog should be or an error (red) not warning (yellow). and let's not fade it out (especially not so quickly). Just leave the alert there.